themes/dark/templates/editShow.mako

```diff
@@ -43,121 +42,133 @@ const startVue = () => {
                     },
                     language: 'en'
                 },
-                defaultEpisodeStatusOptions: [
-                    {text: 'Wanted', value: 'Wanted'},
-                    {text: 'Skipped', value: 'Skipped'},
-                    {text: 'Ignored', value: 'Ignored'}
-                ],
-                seriesLoaded: false,
+                showLoaded: false,
                 saving: false
             }
         },
-        async mounted() {
-            const seriesSlug = $('#series-slug').attr('value');
-            const url = 'series/' + seriesSlug;
-            try {
-                const response = await api.get('series/' + this.seriesSlug);
-                this.series = Object.assign({}, this.series, response.data);
-                this.series.language = response.data.language;
-            } catch (error) {
-                console.debug('Could not get series info for: '+ seriesSlug);
+        computed: Object.assign(store.mapState(['shows']), {
+            params() {
+                return location.search.slice(1).split('&').reduce((obj, pair) => {
```

Comment:
@OmgImAlexis @p0psicles 
A really cool thing I recently found out about:
You can use VueRouter for this, even if it's not a route!
Add `router` to this Vue instance (like you added store above),
then you can use: `this.$route.query` instead of this. :tada: 
![image](https://user-images.githubusercontent.com/10238474/42045796-19ddad5a-7b05-11e8-86f1-c1b9089640f1.png)

Reply:
Ooh cool!

Reply:
Oh I knew that but I didn't want to add the router as we don't need it right now and didn't want to get ahead of myself again.